lib/v2/polkaworld/home.js

```diff
@@ -13,7 +16,7 @@ module.exports = async (ctx) => {
         item: docList.map((item) => ({
             title: item.Title,
             description: item.Title,
-            pubDate: new Date(item.PublishDate),
+            pubDate: timezone(new Date(parseDate(item.PublishDate, 'YYYY-MM-DDTHH:mm:ss.SSSZ')), +8),
```

Comment:
```suggestion
            pubDate: timezone(parseDate(item.PublishDate, 'YYYY-MM-DDTHH:mm:ss.SSSZ'), +8),
```
No need to new Date, parseDate() will return a Date object.